tests/integration/components/flexberry-datetime-picker-test.js

```diff
@@ -32,3 +32,36 @@ test('it renders', function(assert) {
   //Component does not support template block usage.
   assert.equal(this.$().text().trim(), '');
 });
+
+test('Testing dynamicProperties of flexberry-base-component', function (assert) {
+  assert.expect(2);
+
+  let propertyNameValue = 'class';
+  let propertyValueClass = 'firstClass secondClass';
+  let propertyValueClassChange = 'firstClass secondClass thirdClass';
```

Comment:
Values of propertyValueClass & propertyValueClassChange  must be totally different.